src/test/java/DirectoryClientTest.java

```diff
@@ -1,5 +1,5 @@
 import com.aserto.ChannelBuilder;
-import com.aserto.directory.common.v3.ObjectDependency;
+import com.aserto.directory.common.v2.ObjectDependency;
```

Comment:
Where is `v2.ObjectDependency` used? I think we can remove this import. I don't think we want any v2 imports in this test.

Reply:
It was a leftover, it is not used anywhere, removed it.